pandapower/converter/cim/cim2pp/converter_classes/transformers/powerTransformersCim16.py

```diff
@@ -118,6 +120,12 @@ def _create_trafo_characteristics(self, trafo_type, trafo_df_origin):
             trafo_df['r_lv'] = trafo_df['r_lv'] * (1 + trafo_df['r_dev_lv'] / 100)
             trafo_df['x'] = trafo_df['x'] + trafo_df['x'] * trafo_df['x_dev'] / 100
             trafo_df['x_lv'] = trafo_df['x_lv'] * (1 + trafo_df['x_dev_lv'] / 100)
+            fillna_list = ['ratio_dev', 'angle_dev']
+            for one_item in fillna_list:
+                trafo_df[one_item] = trafo_df[one_item].fillna(trafo_df[one_item + '_lv'])
+            trafo_df['ratio_dev'] = trafo_df['ratio_dev'].fillna(1.)
```

Comment:
default for ratio_dev is 1.? Please check. Sry I'm just working in the github explorer today and cannot check the definition of ratio_dev, but assuming dev is deviation I became sceptical.

Reply:
This is just following the existing logic with r_dev and x_dev parameters. This does not refer to deviation, just a development (temporary) parameter definition as far as I understood.